docs/getting-started.md

```diff
@@ -31,6 +31,10 @@ IMAGE_INSTALL_append_pn-acrn-image-sato = " core-image-weston-package"
 
 # set preferred kernel for sos
 PREFERRED_PROVIDER_virtual/kernel = "linux-intel-acrn-sos"
+
+# set kickstart file that affect sos image only
+WKS_FILE_pn-acrn-image-base = "acrn-bootdisk-microcode.wks.in"
+
```

Comment:
All the `acrn-image-*` recipes are SOS images, so can't this be set in the recipes (like they used to `inherit image-acrn`)

Reply:
i tried it, weak assign in recipes not able to override the value assigned in meta-intel. 
hardcode in recipe will not allow local.conf to make further changes.

Reply:
You can send the patch to meta-intel having below for intel-corei7-64 machine which will alow override. 

WKS_FILE ??= "${@bb.utils.contains_any("EFI_PROVIDER", "systemd-boot", "systemd-bootdisk-microcode.wks.in", "grub-bootdisk-microcode.wks.in", d)}"

File location:
http://git.yoctoproject.org/cgit/cgit.cgi/meta-intel/tree/conf/machine/intel-corei7-64.conf

Reply:
there is a default WKS_FILE with "??=" in oe-core, where setting "??=" in meta-intel not able to override it.
https://git.yoctoproject.org/cgit/cgit.cgi/poky/tree/meta/classes/image_types_wic.bbclass#n11

Reply:
set WKS_FILE in  local.conf were not able to override the WKS_FILE set in image recipe, but WKS_FILE_pn-<image name> in local.conf still works. i had made the changes in the new push.